corporate/views/audit_logs.py

```diff
@@ -0,0 +1,102 @@
+from typing import Any
+
+from django.http import HttpRequest, HttpResponse, HttpResponseNotFound
+from django.shortcuts import render
+
+from corporate.lib.activity import ActivityHeaderEntry, format_optional_datetime, make_table
+from zerver.decorator import require_server_admin
+from zerver.lib.typed_endpoint import PathOnly
+from zerver.models.realm_audit_logs import AbstractRealmAuditLog, AuditLogEventType
+from zilencer.models import RemoteRealmAuditLog, RemoteZulipServer, RemoteZulipServerAuditLog
+
+USER_ROLES_KEY = "100: owner, 200: admin, 300: moderator, 400: member, 600: guest"
```

Comment:
This will likely be useful in other audit log views that we implement for realms/users.